echo.go

```diff
@@ -128,20 +128,6 @@ type (
 	}
 )
 
-// HTTP methods
```

Comment:
Wouldn't it break code for users? Perhaps we can mark it deprecated and drop it in a future release.

Reply:
If doubt anyone uses echo.Get instead of http.MethodGet.

Also it's quite easy to replace it if it becomes breaking change for someone.

This change simplifies the codebase by reusing constants from stdlib.

Reply:
Indeed it's very quick to fix, but it's used in the documentation; https://echo.labstack.com/middleware/cors.
I used those constants, rather than the stdlib ones, because of the docs. I know it's a little late, but I can't be the only one getting broken code from this change.

Reply:
@nipemgroup I have update the docs to reflect this change.